src/actions/tokens.go

```diff
@@ -0,0 +1,31 @@
+package actions
```

Comment:
I think the go files needs to be go fmt'd

Reply:
Yes thank you! I noticed my fmt on save wasn't working :/